package.json

```diff
@@ -11,33 +11,26 @@
     "test": "cd theme && jest"
   },
   "dependencies": {
-    "@nuxtjs/style-resources": "1.0.0",
+    "@nuxtjs/style-resources": "^1.2.1",
     "@spryker-vsf/api": "0.6.0-dev.4",
     "@spryker-vsf/catalog-search-suggestions": "0.2.0-dev.1",
     "@spryker-vsf/composables": "0.6.0-dev.4",
     "@spryker-vsf/oidc": "0.1.5-dev.2",
     "@spryker-vsf/oidc-client": "0.2.0-dev.4",
-    "@storefront-ui/vue": "0.10.5",
-    "@vue-storefront/middleware": "2.3.1",
-    "@vue-storefront/nuxt": "2.3.1",
-    "@vue-storefront/nuxt-theme": "2.3.1",
-    "cookie-universal-nuxt": "2.1.4",
-    "core-js": "2.6.12",
-    "dotenv": "10.0.0",
-    "nuxt": "2.14.12",
-    "nuxt-i18n": "6.20.0",
-    "vee-validate": "3.4.5",
-    "vue-scrollto": "2.20.0"
+    "@storefront-ui/vue": "~0.10.5",
+    "@vue-storefront/middleware": "~2.3.4",
+    "@vue-storefront/nuxt": "~2.3.4",
+    "@vue-storefront/nuxt-theme": "~2.3.4",
+    "cookie-universal-nuxt": "^2.1.5",
+    "core-js": "^2.6.12",
```

Comment:
All versions with caret were updated to the latest available version, accept **core-js** which was updated to the latest v.2 version to avoid breaking changes